src/fund/utils/getPopupSize.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  getWindowDimensions,
+  type popupSizes,
+} from '../../internal/utils/getWindowDimensions';
+import { ONRAMP_POPUP_HEIGHT, ONRAMP_POPUP_WIDTH } from '../constants';
+import { ONRAMP_BUY_URL } from '../constants';
+
+type PopupSize = {
+  height: number;
+  width: number;
+};
+
+/**
+ * Gets the appropriate popup dimensions for the given size and funding URL.
+ */
+export function getPopupSize(
```

Comment:
since this is specific to funding, should this be getFundingPopupSize?

Reply:
@alessey Yeah sure I can do that! I figured that having it under `/fund` would make including fund in the name redundant, but probably better to be explicit.

Reply:
@alessey rename done.